<launch>
	<!--<remap from="/camera/odom/sample" to="/mavros/odometry/out" /> -->
	<include file="$(find realsense2_camera)/launch/rs_t265.launch" >
		<arg name="enable_gyro"         value="false"/>
		<arg name="enable_accel"        value="false"/>
		<arg name="enable_pose"         value="true"/>
		<arg name="tf_prefix"           value="t265"/>
		<arg name="publish_odom_tf"     value="true"/>
		<arg name="linear_accel_cov"    value="0.01"/>
  	</include>
	
	<include file="$(find px4_interface)/launch/px4/px4.launch">
		<arg name="fcu_url"             value="/dev/ttyACM1:57600" />   <!-- /dev/ttyACM0:57600 -->
		<arg name="respawn_mavros"      value="false" />
	</include>
	
	<node pkg="tf" type="static_transform_publisher" name="camera_installation" args="0 0 0 0 0 0 base_link_frd t265_odom_frame 1000"/>
	<node name="px4_interface_node" pkg="px4_interface" type="px4_interface_node" output="screen" >
    	<param name="cam_odom_topic"    type="string"       value="/camera/odom/sample"/>
		<param name="world_frame"       type="string"       value="odom_ned"/>
		<param name="body_frame"        type="string"       value="t265_odom_frame"/>
  	</node>

</launch>
